server/src/main/java/org/spine3/server/Entity.java

```diff
@@ -50,6 +59,8 @@
 
     private int version;
 
+    private final M defaultState;
```

Comment:
Yes, I've tried to declare it static, but there is a [restriction...](https://docs.oracle.com/javase/tutorial/java/generics/restrictions.html#createStatic)